torch/csrc/jit/codegen/cuda/codegen.cpp

```diff
@@ -1077,7 +1158,10 @@ class CudaKernelGenerator : private kir::IrVisitor {
 
   void visit(const kir::ForLoop* node) final {
     // TODO(kir): handle this during lowering
-    if (node->iter_domain()->isBroadcast()) {
+    // MMA loops are implicitly handled by mma instructions
+    //  so the for loop itself should be omitted.
+    if (node->iter_domain()->isBroadcast() ||
+        node->iter_domain()->isMMALoop()) {
```

Comment:
As I mentioned in lower_mma.cpp, if we change this part of code like below, I think we could remove `forceVectorize`.

```
 if (node->iter_domain()->isBroadcast()) {
      handleScope(node->body());
      return;
    } else if (node->vectorize() || node->iter_domain()->isMMALoop()) {
      vectorize_scope_ = node->vectorize();
      handleScope(node->body());
      vectorize_scope_ = false;
      return;
    }
```

We would also need to set `vectorize_` flag of `kir::ForLoop`.